Library/Homebrew/test/formula_installer_bottle_spec.rb

```diff
@@ -72,6 +73,40 @@ def temporarily_install_bottle(formula)
     end
   end
 
+  specify "basic bottle install with cellar information on sha256 line" do
+    allow(DevelopmentTools).to receive(:installed?).and_return(false)
+    Homebrew.install_args.parse(["testball_bottle_cellar"])
+    temporarily_install_bottle(TestballBottle.new) do |f|
+      # Copied directly from formula_installer_spec.rb
+      # as we expect the same behavior.
+
+      # Test that things made it into the Keg
+      expect(f.bin).to be_a_directory
+
+      expect(f.libexec).to be_a_directory
+
+      expect(f.prefix/"main.c").not_to exist
+
+      # Test that things made it into the Cellar
+      keg = Keg.new f.prefix
+      keg.link
+
+      bin = HOMEBREW_PREFIX/"bin"
+      expect(bin).to be_a_directory
+
+      expect(f.libexec).to be_a_directory
+
+      skip_relocation = if OS.linux?
+        # skip_relocation is always false on linux
+        false
```

Comment:
Can this call out to a function that defines this?

Reply:
@iMichka ping on this.

Reply:
I tried that but I am unsure how to do that.

I can call `require "extend/os/linux/software_spec"` and import the linux version of BottleSpecification, but this will actually override BottleSpecification for the whole spec. Can I require BottleSpecification from the linux file, with a different class name?

Reply:
I think I got it to work by requiring `extend/os/linux/software_spec` only on linux, and calling directly the method from the overriden BottleSpecification. Let's see if that works.